lms/static/scripts/frontend_apps/components/ErrorDisplay.tsx

```diff
@@ -116,6 +118,8 @@ export default function ErrorDisplay({
 }: ErrorDisplayProps) {
   const message = formatErrorMessage(error, /* prefix */ description);
 
+  const { debug } = useConfig(['debug']);
```

Comment:
Can you just yolo-useConfig from everywhere?

Reply:
Yep 😃

Reply:
`useConfig` is built on `useContext`, which is a way that a component anywhere in the tree can get at data as long as there is a matching "provider" component higher up the tree. This is something like environment variables in a process tree.